internal/pkg/controller/controller_test.go

```diff
@@ -0,0 +1,244 @@
+package controller
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stakater/Reloader/pkg/kube"
+	"k8s.io/api/core/v1"
+	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	client, _           = kube.GetClient()
```

Comment:
Either properly check that the client is successfully created if it's global, or created it in each test and check for error